artemis/base_pytest.py

```diff
@@ -262,6 +262,13 @@ def journey(self, _from, to, datetime,
         for k, v in six.iteritems(kwargs):
             query = "{query}&{k}={v}".format(query=query, k=k, v=v)
 
+        # Override scenario
+        if self.__class__.data_sets[0].scenario in ['distributed', 'experimental', 'asgard']:
+            overridden_scenario = 'experimental'
```

Comment:
```suggestion
            overridden_scenario = 'distributed'
```
experimental is currently an alias to distributed, but we may want to reuse it later for some experimental feature :)